cmake/HalideGeneratorHelpers.cmake

```diff
@@ -718,8 +718,8 @@ function(_Halide_target_link_gpu_libs TARGET VISIBILITY)
     endif ()
 
     if ("${ARGN}" MATCHES "webgpu")
-        if (WEBGPU_NATIVE_LIB)
-            target_link_libraries(${TARGET} INTERFACE ${WEBGPU_NATIVE_LIB})
+        if (DEFINED ENV{HL_WEBGPU_NATIVE_LIB})
+            target_link_libraries(${TARGET} INTERFACE $ENV{HL_WEBGPU_NATIVE_LIB})
         endif ()
```

Comment:
There should definitely be some more validation around this. If this library doesn't exist, it should surely be the case that the node version does.

Reply:
Ah, but if this is going in HalideGeneratorHelpers, then the logic to find the webgpu library can't go in `dependencies/...`, it needs to either be placed here or factored into a Find module.